src/platform/plugins/shared/discover/public/components/data_types/logs/service_name_cell.tsx

```diff
@@ -47,7 +49,8 @@ export const getServiceNameCell =
       <ServiceNameBadgeWithActions
         onFilter={actions.addFilter}
         icon={getIcon}
-        value={serviceNameValue}
+        rawValue={serviceNameValue}
+        value={formatter?.convert(serviceNameValue, 'html', { field }) ?? `${serviceNameValue}`}
```

Comment:
This is another instance where I think `formatFieldValue` would be the best choice.